### PR TITLE
Move forward

### PR DESCRIPTION
**Pull request by ahjohannessen**
- adjust grpc service printer to use fs2 1.0 based
   code.

 - make grpc service printer easier to read by introducing
   constants for common types used in signatures.

 - upgrade dependencies.

### CODE REVIEW

**Comment by ahjohannessen** [comment]
@fiadliel WDYT, can we get this merged and publish a release? :)

File: java-runtime/src/main/scala/syntax/ServerBuilderSyntax.scala [hunk 1]
```diff
@@ -36,6 +35,32 @@ final class ServerBuilderOps[A <: ServerBuilder[A]](val builder: ServerBuilder[A
...
+    * down when the stream is complete.  Shutdown is as follows:
+    *
+    * 1. We request an orderly shutdown, allowing preexisting calls to continue
+    *    without accepting new calls.
+    * 2. We block for up to 30 seconds on termination, using the blocking context
+    * 3. If the server is not yet terminated, we trigger a forceful shutdown
+    *
+    * For different tradeoffs in shutdown behavior, see {{streamWithShutdown}}.
+    */
+  @deprecated("Use resource", "0.4.0")
```
Comment (rossabaker) [inline java-runtime/src/main/scala/syntax/ServerBuilderSyntax.scala]: In http4s, we provide both `stream` and `resource` constructors on the builders.  One can be derived from the other, but people like both.

**Comment by ahjohannessen** [comment]
Nice suggestion @rossabaker - I'll try to use the `invokeCallback` approach from fs2 on another branch to see how it works out.

**Comment by ahjohannessen** [comment]
@rossabaker @fiadliel closing this because of I have a new PR sans `ContextShift[IO]`

**Pull request closed without merging**